engine/rpcserver.go

```diff
@@ -401,20 +401,26 @@ func (s *RPCServer) GetOrderbook(_ context.Context, r *gctrpc.GetOrderbookReques
 	}
 
 	var bids []*gctrpc.OrderbookItem
-	for x := range ob.Bids {
-		bids = append(bids, &gctrpc.OrderbookItem{
-			Amount: ob.Bids[x].Amount,
-			Price:  ob.Bids[x].Price,
-		})
-	}
-
 	var asks []*gctrpc.OrderbookItem
```

Comment:
You can make() these with the len() of ob.Bids / ob.Asks if you want since we append them all anyway and will save having to increase the size of the slice as part of the append()

You can also do assignment  instead of append in the loops below if you really want but the performance gains are minimal/non-existent that its whatever you consider more readable